src/main/resources/application.properties

```diff
@@ -14,6 +14,7 @@ license_conflict=db
 
 # Logger configuration
 logging.pattern.console=%d{dd-MM-yyyy HH:mm:ss.SSS} %magenta([%thread]) %highlight(%-5level) %logger.%M - %msg%n
+spring.profiles.active=singlescan
```

Comment:
This line is not required here.
Also, please modify the next property in `application-singlescan.properties`:
spring.datasource.url=jdbc:h2:mem:singlescan;DATABASE_TO_UPPER=false;MODE=MySQL;NON_KEYWORDS=USER;DB_CLOSE_ON_EXIT=FALSE

Reply:
done